### DISCUSSION TITLE
Welcome to app Discussions!

### DISCUSSION THREAD

**Discussion by samdark** [yiisoft/app#107] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by samdark** [comment]
Overall I wonder how well it works. We have [a forum for discussions](https://forum.yiiframework.com/) and it's worked well so far but let's try GitHub as well...

**Reply by roxblnfk** [reply]
Can we organize multilingual sections here?

**Reply by samdark** [reply]
> Can we organize multilingual sections here?

Let's stick to English for now.

**Reply by mikehaertl** [reply]
Cool, finally a place where you can ask questions in a place that is close to the sources. I'm curious how this will work out.

**Reply by tomaszkane** [reply]
I believe forum is better tool (more features) for communicate / discussions. Github should be code repository and issue tracker.
Hm, what is advantages Discussion over Forum?
Leaving aside better integration with Github?

**Reply by samdark** [reply]
Depends on what discussion is that. If it's about a concrete repo issues or Q&A then, maybe, it could be alright.

**Comment by SOHELAHMED7** [comment]
Hello World,

I'm an independent software engineer with 5+ years of experience. I create web apps and API using PHP, JavaScript, PostgreSQL, Redis and websites & blogs using GatsbyJS. I 💪 to solve real world problems using computer programming.

Professionally Yii is the first framework I learnt and made an app on it. I am keeping watch on Yii 3 and excited for it.

Overall I tend to prefer Forum first for discussion and for technical question and answer, StackOverflow and Forum and then GitHub.

On the Internet, I can be found at 
  - [sohelahmed.site](https://sohelahmed.site/)
 - [GitHub](https://github.com/sohelahmed7)
 - [StackOverflow](https://stackoverflow.com/users/3794786/sohelahmedm)
 - [Twitter](https://twitter.com/SohelAhmedM)
 - [LinkedIn](https://www.linkedin.com/in/sohelahmed/)
